Formula/unrar.rb

```diff
@@ -31,6 +32,15 @@ def install
     system "make", "clean"
     system "make", "lib"
     lib.install shared_library("libunrar")
+
+    prefix.install "license.txt"
+  end
+
+  def caveats
+    <<~EOS
+      The license has been installed at #{opt_prefix}/license.txt. Please ensure that you
+      agree with the terms of the license before using this software.
```

Comment:
Let's pattern this after qt and sslabs-scan
```suggestion
      We agreed to the UnRAR license for you:
        #{opt_prefix}/license.txt
      If this is unacceptable you should uninstall the formula.
```